compal/__init__.py

```diff
@@ -236,6 +244,14 @@ def logout(self):
         """
         return self.xml_setter(Set.LOGOUT, {})
 
+    def set_modem_mode(self):
+        """
+        Sets router to Modem-mode
+        After setting this, router will not be reachable by IP!
+        It needs factory reset to function as a router again!
+        """
+        return self.xml_setter(Set.NAT_MODE, {'NAT': NatMode.off.value})
```

Comment:
I needed this feature. This will probably work on a modem that is not 'activated' yet